src/mpid/ch4/src/ch4r_rma_origin_callbacks.h

```diff
@@ -157,6 +157,20 @@ static inline int MPIDI_CH4U_acc_origin_cmpl_handler(MPIR_Request * sreq)
 }
 
 #undef FUNCNAME
+#define FUNCNAME MPIDI_CH4U_get_acc_origin_cmpl_handler
+#undef FCNAME
+#define FCNAME MPL_QUOTE(FUNCNAME)
+static inline int MPIDI_CH4U_get_acc_origin_cmpl_handler(MPIR_Request * sreq)
+{
+    int mpi_errno = MPI_SUCCESS;
+    MPIR_FUNC_VERBOSE_STATE_DECL(MPID_STATE_CH4U_GET_ACC_TX_HANDLER);
+    MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_CH4U_GET_ACC_TX_HANDLER);
+    MPIDI_Request_complete(sreq);
```

Comment:
Is it because some of them want to decrement window counter for outstanding_ops ?
OPA_decr_int(&MPIDI_CH4U_WIN(win, outstanding_ops));